### ISSUE TITLE
Js perfs

### ISSUE THREAD

**Issue by CosticaPuntaru** [EveryMatrix/workshops#2]
Perf difference between:
If vs switch
For vs array.Foreach

**Comment by fk1blow** [comment]
👍

**Comment by Clokze** [comment]
You really want to go there? I'd much rather do some form of a workshop on profiling and benchmarking, these things tend to get optimized by the VMs and they change quite often (thats why they are usually labeled micro optimizations).

**Comment by fk1blow** [comment]
yeah, +1 for the profiling and bencgmarking because of the reasons outlined above
